categories.go

```diff
@@ -219,7 +227,7 @@ func getCategoriesOutput(ctx context.Context, categories map[string]*packages.Ca
 	}
 	sort.Strings(keys)
 
-	var outputCategories []*packages.Category
+	outputCategories := []*packages.Category{}
```

Comment:
Unrelated change. It changes the response of the `/categories` endpoint to generate `[]` instead of `null` when no packages satisfy the conditions.